packages/next/src/server/load-components.ts

```diff
@@ -147,13 +155,20 @@ async function loadComponentsImpl<N = any>({
   const [
     buildManifest,
     reactLoadableManifest,
+    dynamicCssManifest,
     clientReferenceManifest,
     serverActionsManifest,
   ] = await Promise.all([
     loadManifestWithRetries<BuildManifest>(join(distDir, BUILD_MANIFEST)),
     loadManifestWithRetries<ReactLoadableManifest>(
       join(distDir, REACT_LOADABLE_MANIFEST)
     ),
+    // This manifest will only exist in Pages dir && Production && Webpack.
+    isAppPath || process.env.TURBOPACK
+      ? undefined
+      : loadManifestWithRetries<DynamicCssManifest>(
+          join(distDir, `${DYNAMIC_CSS_MANIFEST}.json`)
```

Comment:
We need to add this as a required file for standalone mode https://github.com/vercel/next.js/blob/canary/packages/next/src/build/index.ts#L2616